core-java-modules/ParsingDates/src/main/java/com/parse/parsingdates/SimpleDateUtils.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.parse.parsingdates;
+
+import java.text.ParseException;
+import org.apache.commons.lang3.time.DateUtils;
+
+public class SimpleDateUtils {
+
+    public boolean parseDate(String date) {

```

Comment:
Same here, I'm thinking it's better to return the parsed date, and not coerce it down to a boolean.